### ISSUE TITLE
Incorrect Session->getAttributes() tests

### ISSUE THREAD

**Issue by piotras** [phpcr/phpcr-api-tests#31]
I might be wrong, but two SessionReadMethodsTest tests seems to be incorrect:
- testGetAttributeNames()
- testGetAttribute()

What I understood, is the fact that both of them, test if Credentials object is the same instance available from ImplementationLoader and Session. Which of course doesn't have to correct, cause Credentials is implementation specific and it's not guaranteed that getAttribute() method will return the same object which is held by ImplementationLoader.
In other words test assumes that Credentials is static object and both Session and loader hold the same reference to its instance.
